source/game.production.bmx

```diff
@@ -452,6 +446,7 @@ Type TProduction Extends TOwnedGameObject
 
 		productionStep = TVTProductionStep.SHOOTING
 
+		'TODO !!!diese Fixes könnten zu spät sein für eine-Live-Produktion, die tatsächlich zum Live-Zeitpunkt gesendet wird
```

Comment:
Ich zieh diesen Kommentar mal "raus", damit er im PR auch direkt verlinkt ist.

Frage: inwieweit ist das ein Problem? In dem Moment, wo die Live-Ausstrahlung "startet" wird die Funktion aufgerufen (indirekt ueber main.bmx onminute -> GetProductionManager().Update()). Und zwar bevor die Zuschauerzahlen berechnet werden (die ja dann direkt auf die Produktionswerte zugreifen - auch wenn man bei "live" wohl eher von "erwarteten Produktionswerten" reden muesste - aber wir wollen es denke ich lieber einfach halten).

Reply:
Ich bin nur über diese Stelle gestolpert und hatte mich gefragt, ob es ein Problem sein könnte. Aus der Antwort bin ich mir aber noch immer nicht ganz sicher:
Die Zuschauerzahlberechnung hat dann schon die aktualisierten Werte zur Verfügung, korrekt? Denn ansonsten könnte es fies sein (Always-Live später gesendet schneidet wegen der schon aktualisierten Werte besser ab als "unmittelbar" live gesendet).